File: Tarea 7/python/Productos.py

cantidad = 5
articulos = []
Precios = []
total = 0
for i in range(cantidad):
    art = str(input(f"Dame el articulo {i+1}: "))
    articulos.append(art)
    try:
        pre = float(input(f"¿Cual es el precio del articulo -{articulos[i]}- ?: $"))
        Precios.append(pre)
        total = total + pre
    except:
        print("vuvelve a intentarlo, por favor solo dame numeros")
        exit()

print("-----------------------------")
for i in range(cantidad):   
    print(articulos[i], "------ $" , end="")
    print(Precios[i])
print("-----------------------------")
print("Total a pagar: $", total)

File: Tarea 7/c++/Productos.cpp
#include <iostream>
using namespace std;
int main(){   
    float total = 0;
    int cantidad = 5;
    string articulo[cantidad];
   
    float precio[cantidad];
    for(int i = 0; i<cantidad;i++){
        cout << "Dame el articulo " << i+1 << ": ";
        cin >> articulo[i];
        cout << "¿Cual es el precio del articulo -" << articulo[i]<<"- ?: $";
        cin >> precio[i];
        total = total + precio[i];
    }
    cout << "-----------------------------" << endl;
    for (int i = 0; i < cantidad; i++){
        cout << articulo[i] << "------ $" << precio[i] << endl;
    }
    cout << "-----------------------------" <<endl;
    cout <<"Total a pagar: $" << total << endl;
}